### PR TITLE
SW-1585 Clean up Edit buttons and some newer Delete buttons

### PR DESCRIPTION
**Pull request by alexTerraformation**
Add pencil icon to all edit buttons and make edit buttons only pencil icon on mobile (except for empty states).  Also add trash can icon to some newer delete buttons from seeds v2.
Screen Shot 2022-10-04 at 8 50 21 PM
Screen Shot 2022-10-04 at 8 51 00 PM
Screen Shot 2022-10-04 at 8 51 11 PM
Screen Shot 2022-10-04 at 8 51 20 PM
Screen Shot 2022-10-04 at 8 51 41 PM
Screen Shot 2022-10-04 at 8 51 47 PM
Screen Shot 2022-10-04 at 8 52 27 PM
Screen Shot 2022-10-04 at 8 52 36 PM
Screen Shot 2022-10-04 at 8 53 57 PM
Screen Shot 2022-10-04 at 8 55 00 PM
Screen Shot 2022-10-04 at 8 55 10 PM
Screen Shot 2022-10-04 at 8 55 22 PM
Screen Shot 2022-10-04 at 8 55 52 PM

### CODE REVIEW

**Comment by rachel-tf-design** [comment]
Looks great! Thanks @alexTerraformation :)

**Review by constanzauanini — APPROVED** [review]

File: src/components/Organization/index.tsx [hunk 1]
```diff
@@ -109,11 +109,11 @@ export default function OrganizationView({ organization }: OrganizationViewProps
...
-            <Button
-              label={isMobile ? strings.EDIT : strings.EDIT_ORGANIZATION}
-              priority='secondary'
-              onClick={goToEditOrganization}
-            />
+            {isMobile ? (
+              <Button icon='iconEdit' priority='secondary' onClick={goToEditOrganization} />
+            ) : (
+              <Button icon='iconEdit' label={strings.EDIT} priority='secondary' onClick={goToEditOrganization} />
+            )}
```
Comment (karthikbtf) [inline src/components/Organization/index.tsx:116]: Do we have a preference on how to handle cases like this - where the only difference is the label?
```
{ isMobile ? <mobile markup> : <desktop markup>}
```
vs
```
<markup label={isMobile ? '' : string} >
```

We have been mostly using the latter in the codebase.. however, the former does express intent clearly.

Just a question for all. Not requesting any changes at this point.

Comment (constanzauanini) [reply src/components/Organization/index.tsx:116]: What I try to do is that if only one attribute is different between the two elements option 2 is better, but if we have more than one attributed changed (for example label, priority, icon) it's better to create two elements

Comment (charlie-tf) [reply src/components/Organization/index.tsx:116]: I like the latter myself, it's more concise and also gets the intent across I think. Certainly if you have a lot of props conditional on `isMobile` the former would be the way to go. If you had other props that were conditional on other variables you might get into some kinda nasty nesting, but at that point you might want to rethink inlining the logic and maybe break out into another component 🤷

Comment (karthikbtf) [reply src/components/Organization/index.tsx:116]: I am leaning towards the former now that I see it in code. We have an impasse 😱 - we could evolve our codestyle over time maybe, now that the team is growing?

**Pull request merged**